agent/reconciler/host_reconciler.go

```diff
@@ -206,6 +212,7 @@ func (r *HostReconciler) hostCleanUp(ctx context.Context, byoHost *infrastructur
 		}
 		err = bundleInstaller.Uninstall(k8sVersion, byohBundleTag)
 		if err != nil {
+			logger.Info(err.Error())
```

Comment:
We are already returning `err` (which will be logged at some point in the error handling chain), so no need to log separately.